### PR TITLE
Dispatch logout action when API calls return 401 or 403 errors

### PR DESCRIPTION
**Pull request by adamstankiewicz**
When an API response returns an error with a status code of 401 (unauthorized) or 403 (forbidden), we should force logout the user so they return to the login page to reset their session.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -24,7 +28,17 @@ const getAccessToken = () => {
...
+axios.interceptors.response.use(response => response, (error) => {
+  const errorStatus = error.response.status;
+  if (errorStatus === 401 || errorStatus === 403) {
+    store.dispatch(logout());
+  }
+  return Promise.reject(error);
+});
+
 export {
+  axios,
```
Comment (georgebabey) [inline src/utils.js]: I think we should call this something different, maybe `httpClient` so we know its not stock axios

File: config/webpack.dev.config.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ module.exports = Merge.smart(commonConfig, {
       NODE_ENV: 'development',
       LMS_BASE_URL: 'http://localhost:18000',
       DATA_API_BASE_URL: 'http://localhost:8000',
-      LMS_CLIENT_ID: '6cb72dc5675a1e7efe81',
+      LMS_CLIENT_ID: 'CMehRRNqfiBRVJKnPOkjBDjAvurtnHpELoehKAvZ',
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply config/webpack.dev.config.js]: Haha yeah, I shook my head when I accidentally committed that after just having that conversation :P

File: src/utils.js (hunk 1 above)
Comment (adamstankiewicz) [reply src/utils.js]: Good idea!

File: src/utils.js [hunk 3]
```diff
@@ -30,15 +30,15 @@ const getAccessToken = () => {
...
+  const errorStatus = error && error.response && error.response.status;
   if (errorStatus === 401 || errorStatus === 403) {
     store.dispatch(logout());
   }
   return Promise.reject(error);
 });
 
 export {
-  axios,
+  axios as httpClient,
```
Comment (georgebabey) [inline src/utils.js]: I feel like this should be moved into its own file.

**Pull request merged**